airlock_processor/StatusChangedQueueTrigger/__init__.py

```diff
@@ -18,7 +18,7 @@ class RequestProperties(BaseModel):
     new_status: str
     previous_status: Optional[str]
     type: str
-    workspace_id: str
+    unique_identifier_suffix: str
```

Comment:
Across much of this change, isn't the _unique_identifier_suffix_ always the **workspace** one?
If so, shouldn't we call it like that?

Reply:
changed